bin/compile

```diff
@@ -247,8 +287,9 @@ build_dependencies() {
 
   cache_status="$(get_cache_status "$CACHE_DIR")"
   start=$(nowms)
-
-  if $YARN; then
+  if [[ "$YARN2" == "true" ]]; then
+    echo "Using Yarn 2, skipping installations."
```

Comment:
`yarn install` would still need to run for dependencies with a postinstall script when using PnP and to create the `node_modules` folder when using the `node-modules` linker

Reply:
Right - thanks for pointing this out. The assumption is that the install scripts will not be run for now since install is not run. I made a note in the PR description. We will eventually want to support this, of course, but not in this first draft.

Reply:
Alright, keep in mind any dependency with a postinstall script wont work. So for example `core-js` wont be found

Reply:
Yeah, we'll be working on that and keep that in mind. This is a priority. Thanks for the review!